pr_review_list.rb

```diff
@@ -113,3 +115,4 @@ def does_array_have_pr(array, pr_number)
 File.open('report.json', 'wb') do |f|
   JSON.dump(open_prs, f)
 end
+end
```

Comment:
formatting/indentation seems off, are we using rubocop

Reply:
No @tphoney

Reply:
I believe @tphoney is suggesting that you should be using Rubocop to ensure that code formatting violations like this are caught. E.g. `bundle exec rake rubocop`

Reply:
I will run this command in the terminal tomorrow, thanks @tphoney and @sanfrancrisko